tox.ini

```diff
@@ -22,6 +22,13 @@ passenv =
     CI
     TRAVIS
     TRAVIS_*
+    API_KEY_EIA
+
+[testenv:mcoe_temp]
+skip_install = false
+commands =
+    pytest test/validate/mcoe_test.py::test_mcoe_self --pudl_out=AUTO --pudl_in=AUTO --live_ferc1_db=AUTO --live_pudl_db=AUTO
```

Comment:
Did you want to leave this all in? Or was it only for  the purpose of testing the EIA API key integration into Travis?

Reply:
ah oops. yes this was just for easier testing. i will remove it